ci/perf_trace_sweep.py

```
@@ -0,0 +1,125 @@
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument("--port", default="COM25", help="serial port (default COM25)")
+    parser.add_argument("--lanes", type=int, default=12, help="number of PARLIO lanes")
+    parser.add_argument("--strip-size", type=int, default=256, help="LEDs per lane")
+    parser.add_argument(
+        "--json", action="store_true", help="emit raw JSON instead of a table"
+    )
+    args = parser.parse_args()
+
+    try:
+        result = asyncio.run(run_sweep(args.port, args.lanes, args.strip_size))
+    except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)